common/os_calls.c

```diff
@@ -3307,6 +3307,17 @@ g_setsid(void)
 #endif
 }
 
+/*****************************************************************************/
+int
+g_getusername(char *name, unsigned int len)
```

Comment:
I'm fine with this function name. BTW, why is this named `getusername` rather than `getlogin`?

Reply:
Good point - I think it was where I was coming from when I was putting the calling function together. I agree that `g_getlogin` is more expected here, so I've changed it.